lib/spark.js

```diff
@@ -257,20 +257,17 @@ export default class Spark {
       const isHealthy = this.#activity.isHealthy()
 
       let delay
-      if (isHealthy) {
-        delay = calculateDelayBeforeNextTask({
-          roundLengthInMs: APPROX_ROUND_LENGTH_IN_MS,
-          maxJitterInMs: MAX_JITTER_BETWEEN_TASKS_IN_MS,
-          maxTasksPerRound: this.#tasker.maxTasksPerRound,
-          lastTaskDurationInMs: duration,
-        })
-      } else {
-        delay = OFFLINE_RETRY_DELAY_MS
-      }
+      delay = calculateDelayBeforeNextTask({
+        isHealthy,
```

Comment:
```suggestion
        isHealthy = true,
```

So that we don't have to update all existing tests.

Reply:
Yes, you are right - I meant to apply the default in the function signature 👍🏻